app/src/components/market/sections/market_bond/market_bond.tsx

```diff
@@ -61,8 +64,10 @@ const MarketBondWrapper: React.FC<Props> = (props: Props) => {
   const [isModalTransactionResultOpen, setIsModalTransactionResultOpen] = useState(false)
   const [outcomeIndex, setOutcomeIndex] = useState<number>(0)
   const probabilities = balances.map(balance => balance.probability)
+  const initialBondAmount =
+    networkId === networkIds.XDAI ? parseUnits('10', nativeAsset.decimals) : parseUnits('0.01', nativeAsset.decimals)
   const [bondEthAmount, setBondEthAmount] = useState<BigNumber>(
```

Comment:
Just a nomenclature change, we should probably change the state and setter to [bondNativeAssetAmount, setBondNativeAssetAmount]